packages/admin/config/filament.php

```diff
@@ -207,10 +207,12 @@
         ],
         'sidebar' => [
             'is_collapsible_on_desktop' => false,
+            'has_hamburger_icon_on_desktop' => false,
             'groups' => [
                 'are_collapsible' => true,
             ],
             'width' => null,
+            'collapsed_width' => null,
```

Comment:
Do we need a config key for this, or is it easy to style with CSS using the `filament-main` and `filament-sidebar` hooks?